docs/README.Linux.md

```diff
@@ -24,39 +24,11 @@ Several distribution **[specific build guides](README.md)** are available.
 ## 1. Document conventions
 This guide assumes you are using `terminal`, also known as `console`, `command-line` or simply `cli`. Commands need to be run at the terminal, one at a time and in the provided order.
 
-This is a comment that provides context:
```

Comment:
I think this whole section could be moved to its own .md file and just linked in every document that uses it. I think removing this for a single build guide doesn't make much sense, they all should follow the same structure for consistency.

Reply:
That's a good point. Is there value in this section for the other guides? I assume most folks that reach here know what a command is and how to read tips, etc?

In my case, I found that it got it in the way of obtaining useful information, but perhaps it's useful for other folks?

Reply:
@enen92 Thoughts? Main questions:

1. Is the section about what a command is, what a tip is, etc needed for any of the guides?  If not, I'm happy to remove it from all guides. If it is, I can create a new doc that links to it, and update this PR.

2. Any concerns about the other changes in the PR?

Reply:
I don't see an issue with the other changes, they are quite welcome. Regarding 1) I also don't have strong feelings. People should be familiar enough with docs/markdown if they wish to compile software...
To avoid cluttering this PR I'd keep it as is today and squash the commits into a single one so that it can go in.
In another PR you can try to remove those conventions from all build guides.

Reply:
Sounds good. I've gone ahead and squashed the commits. Unfortunately I messed up midway and it added more reviewers to the PR :) Should've switched to Draft before I did that. Anyhow, I think it's in good shape now.